.github/workflows/tests.yml

```
@@ -0,0 +1,58 @@
+name: Automated Tests
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
```

actionlint (1.7.10)

[error] "jobs" section should not be empty. please remove this section if it's unnecessary

(syntax-check)